<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SigningCertificateSelectionWidget</class>
 <widget class="QWidget" name="SigningCertificateSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>178</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="pgpCombo"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="cmsCombo"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="rememberCO">
     <property name="text">
      <string>Remember these as default for future operations</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="pgpLabel">
     <property name="text">
      <string>OpenPGP Signing Certificate:</string>
     </property>
     <property name="buddy">
      <cstring>pgpCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="cmsLabel">
     <property name="text">
      <string>S/MIME Signing Certificate:</string>
     </property>
     <property name="buddy">
      <cstring>cmsCombo</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
